src/main/webapp/Portal.html

```diff
@@ -138,9 +138,6 @@ <h2 class="center" style="color:white; padding-top: 180px">
 			<div id="alertWrapper">
 				<div id="alertContainer"></div>
 			</div>
-			<div id="headerPanel"></div>
-			<div id="rootPanel"></div>
-			<div id="footerPanel"></div>
 			<div id="initialLoadingUI" style="padding-top: 180px">
```

Comment:
if there's a problem with the initial loading UI being above the rest of the content, let's just remove it from the DOM after load!